CHANGELOG.rst

```diff
@@ -512,15 +512,15 @@ Improved Documentation
   should be treated in functions that accept |array_like| or |Quantity|
   inputs. (:pr:`1673`)
 - Added an admonition to the |changelog guide| that describes how to
-  change reST_ links for removed code objects into inline literals in old
+  change |reStructuredText| links for removed code objects into inline literals in old
```

Comment:
I decided to spell out reStructuredText in full because the "reST" acronym is too ambiguous with REST API, and the word "rest" is a lot less searchable than spelling it out.